pulsar/internal/host_resolve.go

```diff
@@ -0,0 +1,107 @@
+package internal
```

Comment:
Apache license should be included at the beginning of every file?

Reply:
Yes, I am fixing the errors in CI.